pkg/utils/filewatcher_test.go

```diff
@@ -13,32 +13,32 @@ import (
 func TestFileWatcher(t *testing.T) {
 	dir := t.TempDir()
 	path := filepath.Join(dir, "file.txt")
-	os.WriteFile(path, []byte("1"), 0o644)
+	_ = os.WriteFile(path, []byte("1"), 0o600)
 
 	fw, err := NewFileWatcher(path)
 	fw.writeGracePeriod = 50 * time.Millisecond // reduce the delay to make the test runs faster.
 	assert.NoError(t, err)
-	fw.w.Add(path)
+	_ = fw.w.Add(path)
```

Comment:
since this is new code added by this PR, this could also be squashed in the commit adding this code.

Reply:
my bad, done!